habitat/tasks/rearrange/rearrange_sensors.py

```diff
@@ -0,0 +1,601 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import magnum as mn
+import numpy as np
+from gym import spaces
+
+from habitat.core.embodied_task import Measure
+from habitat.core.registry import registry
+from habitat.core.simulator import Sensor, SensorTypes
+from habitat.sims.habitat_simulator.habitat_simulator import (
+    HabitatSimDepthSensor,
+    HabitatSimRGBSensor,
+)
+from habitat.tasks.nav.nav import PointGoalSensor
+from habitat.tasks.rearrange.rearrange_sim import RearrangeSim
+from habitat.tasks.utils import get_angle
+
+# TODO: @maksymets should be accessed through Robot API
+EE_GRIPPER_OFFSET = mn.Vector3(0.08, 0, 0)
+
+
+@registry.register_sensor
+class HeadRgbSensor(HabitatSimRGBSensor):
+    def _get_uuid(self, *args, **kwargs):
+        return "robot_head_rgb"
+
+
+@registry.register_sensor
+class HeadDepthSensor(HabitatSimDepthSensor):
+    def _get_uuid(self, *args, **kwargs):
+        return "robot_head_depth"
+
+
+@registry.register_sensor
+class ArmRgbSensor(HabitatSimRGBSensor):
+    def _get_uuid(self, *args, **kwargs):
+        return "arm_rgb"
+
+
+@registry.register_sensor
+class ArmDepthSensor(HabitatSimDepthSensor):
+    def _get_uuid(self, *args, **kwargs):
+        return "arm_depth"
+
+
+@registry.register_sensor
+class ThirdRgbSensor(HabitatSimRGBSensor):
```

Comment:
This is really unnecessary. UUIDs are defined in the configs. These sensor can be defined from raw onfigs and don't warrant a sensor type